src/controllers/browser.ts

```diff
@@ -15,21 +15,27 @@ export async function initWhatsapp(session: string) {
 
 export async function injectApi(page: puppeteer.Page) {
   await page.addScriptTag({
-    path: require.resolve(path.join(__dirname, '../lib/wapi', 'wapi.js'))
+    path: require.resolve(path.join(__dirname, '../lib/wapi', 'wapi.js')),
   });
   await page.addScriptTag({
-    path: require.resolve(path.join(__dirname, '../lib/middleware', 'middleware.js'))
+    path: require.resolve(
+      path.join(__dirname, '../lib/middleware', 'middleware.js')
+    ),
   });
 
   return page;
 }
 
 async function initBrowser(session: string) {
```

Comment:
I suggest to add here the possibility to set the `headless` mode and the devTool `puppeteer` options, this is useful when someone wants to debug some problems (maybe some api change that throws an error on console) or simply when sulla is running in a background process and the user cannot see the QR. Based on this I also added an event to get the QR image data

Reply:
Adding